src/gei/Commands/MigrateRepo/MigrateRepoCommandArgs.cs

```diff
@@ -25,6 +25,8 @@ public class MigrateRepoCommandArgs : CommandArgs
         public bool NoSslVerify { get; set; }
         public string GitArchiveUrl { get; set; }
         public string MetadataArchiveUrl { get; set; }
+        public string GitArchiveFilePath { get; set; }
+        public string MetadataArchiveFilePath { get; set; }
```

Comment:
I just check the `bbs2gh migrate-repo` command and noticed that we used `ArchivePath` so for consistency it's better to remove the file form the variable names:
```suggestion
        public string GitArchivePath { get; set; }
        public string MetadataArchivePath { get; set; }
```